test/MonthReportMakerTest.java

```diff
@@ -1,162 +0,0 @@
-import ControllersPresenters.ReportLevel;
```

Comment:
Merge main so that you don't delete the Tests here.

Reply:
thank you, will do!